### PR TITLE
Added description of activation poll method.

### PR DESCRIPTION
**Pull request by nikolaystoilov-okta**
Updated the documentation with sample of the poll endpoint, request and response.
Updated the postman collection with information about the endpoint.

Resolves: OKTA-109889

Reviewers:
@mystiberry-okta @gauthamgowrishankar-okta @tthompson-okta

### CODE REVIEW

**Comment by tthompson-okta** [comment]
Noticed a typo in this area of the doc:

pushfactortypo

Can you fix this while you're here?

File: _source/_docs/api/resources/authn.md [hunk 1]
```diff
@@ -2301,6 +2301,102 @@ curl -v -X POST \
...
+      "vendorName":"OKTA",
+      "_embedded":{
+        "activation":{
+          "expiresAt":"2016-12-22T21:41:47.000Z",
+          "factorResult":"WAITING",
+          "_links":{
+            "send":[
+              {
+                "name":"email",
+                "href":"http://your-domain.okta1.com:1802/api/v1/authn/factors/opfh52xcuft3J4uZc0g3/lifecycle/activate/email",
```
Comment (tthompson-okta) [inline _source/_docs/api/resources/authn.md]: I noticed a bunch of `okta1.com`'s slipped in here, and also in your previous PR with the Duo examples. Can you change all of these to `okta.com`?

File: _source/_docs/api/resources/authn.md [hunk 2]
```diff
@@ -2301,6 +2301,102 @@ curl -v -X POST \
 }
 ~~~
 
+##### Poll for Push Factor Activation
+
+After the push notification is sent to user's device we need to know when the user complete the activation. This is done by polling the "poll" link.
```
Comment (tthompson-okta) [inline _source/_docs/api/resources/authn.md]: Nit: "complete" should be "completes"

File: _source/_docs/api/postman/authentication.json [hunk 3]
```diff
@@ -326,6 +326,26 @@
 			"rawModeData": "{\n  \"stateToken\": \"{{stateToken}}\"\n}"
 		},
 		{
+			"id": "aad66581-a251-c87b-395c-ca8e47892a66",
+			"headers": "Accept: application/json\nContent-Type: application/json\n",
```
Comment (gauthamgowrishankar-okta) [inline _source/_docs/api/postman/authentication.json]: just make sure "\n" is fine on the final page view or else please remove that

File: _source/_docs/api/postman/authentication.json [hunk 4]
```diff
@@ -326,6 +326,26 @@
...
+			"version": 2,
+			"tests": "",
+			"currentHelper": "normal",
+			"helperAttributes": {},
+			"time": 1481758444646,
+			"name": "Poll for Factor enrollment",
+			"description": "",
+			"collectionId": "2100a81e-0e91-ead1-f440-8db8fed2f315",
+			"responses": [],
+			"rawModeData": "{\n  \"stateToken\": \"{{stateToken}}\",\n  \"factorType\": \"web\",\n  \"provider\": \"DUO\"\n}"
```
Comment (gauthamgowrishankar-okta) [inline _source/_docs/api/postman/authentication.json]: same comment as above

**Comment by gauthamgowrishankar-okta** [comment]
couple of nits else LGTM 🚀

File: _source/_docs/api/postman/authentication.json (hunk 3 above)
Comment (nikolaystoilov-okta) [inline _source/_docs/api/postman/authentication.json]: Those changes are part of the exported json from postman. They are not visible in the pages. I think they are used as dividers.

**Comment by mystiberry-okta** [comment]
This will show up on the website next Thursday. Merging now.

**Pull request merged**